OneCommunityGlobal/HighestGoodNetworkApp
# Sheetal M - Add unit test for monthlyDashboardDataReducer.js

# Description
Unit test for monthlyDashboardDataReducer

## Related PRS (if any):
No related PRs

## Main changes explained:
 - Added a new file under reducers/tests called monthlyDashboardDataReducer.test.js
 - The list of test cases for monthlyDashboardDataReducer
   - return the default value for monthly dashboard data  when no action is specified
   - handles GET_MONTHLY_DASHBOARD_DATA action by returning payload  

## How to test:
1. Checkout the current branch
2. Run npm test monthlyDashboardDataReducer.test.js
3. Verify all test cases pass without any errors

## Screenshots or videos of changes:
![monthlyDashboardDataReducer-testsuite](https://github.com/user-attachments/assets/10b680c1-e638-4c47-912e-91501c539117)
